co2eq/purchase/index.js

```diff
@@ -105,25 +110,46 @@ export function modelCanRun(activity) {
 
   return false;
 }
+
 function correctWithParticipants(footprint, participants) {
   return footprint / (participants || 1);
 }
+
 function extractEur({ costAmount, costCurrency }) {
   return (costAmount && costCurrency)
     ? convertToEuro(costAmount, costCurrency)
     : null;
 }
 
+function CPIConversion(eurAmount, countryCodeISO2, datetime) {
+  if ((!eurAmount) || (!datetime)) {
+    return eurAmount;
+  }
+  const countryIndicator = countryCodeISO2 || AVERAGE_CPI_COUNTRY_INDICATOR;
+  const coicopIndicator = AVERAGE_CPI_COICOPCODE_INDICATOR;
+
+  const currentDateIndicator = datetime.getFullYear()
+  const CPIcurrent = CPI_BY_COUNTRY[countryIndicator][coicopIndicator][currentDateIndicator]
+  const CPI2011 = CPI_BY_COUNTRY[countryIndicator][coicopIndicator]['2011']
```

Comment:
instead of assuming the values are always 2011, you should look up in footprint.yml the `year` key.
If the year key is not present, then I would throw an error

Reply:
Oh yes good idead! I did not know that it was there :)